.github/actions/setup-windows/action.yml

```diff
@@ -29,8 +29,17 @@ runs:
     - name: Enable long paths on Windows
       shell: powershell
       run: |
-        Set-ItemProperty -Path "HKLM:\\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1
-
+        if(-not (Test-Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem")) {
+          New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force
```

Comment:
Just a note, if this works, maybe it would also help fixing the build issue @jbschlosser reported on TreeHugger last week, i.e.

```
c1xx: fatal error C1081: 'C:\actions-runner\_work\pytorch\pytorch\test\cpp_extensions\libtorch_agnostic_extension\build\temp.win-amd64-cpython-39\Release\actions-runner\_work\pytorch\pytorch\test\cpp_extensions\libtorch_agnostic_extension\libtorch_agnostic\csrc\libtorch_agnostic_kernel.obj': file name too long
```